ra/registration-authority_test.go

```diff
@@ -317,8 +317,8 @@ func TestValidateEmail(t *testing.T) {
 	}{
 		{"an email`", unparseableEmailDetail},
 		{"a@always.invalid", emptyDNSResponseDetail},
-		{"a@always.timeout", "DNS query timed out"},
-		{"a@always.error", "DNS networking error"},
+		{"a@always.timeout", "DNS query timed out during A-record lookup of always.timeout"},
```

Comment:
Ah. Without consulting the implementation of `always.timeout` I had guessed any question would timeout, not just `A` ones. Thanks!